### DISCUSSION TITLE
Querying twitter with SPARQL and then SQL

### DISCUSSION THREAD

**Discussion by justin2004** [SPARQL-Anything/sparql.anything#178] [General]
I saw this and it reminded me of SPARQL Anything:

https://twitter.com/kidehen/status/1461061290198192136?s=20

**Comment by enridaga** [comment]
Virtuoso's SPARQL+SQL feature is really nice, I used it in several projects in the past. With SPARQL Anything, Twitter data can be queried via a Web API call, a bit like what you have demonstrated in https://github.com/justin2004/weblog/tree/master/dynamic_pagination_with_sparql_anything

**Reply by justin2004** [reply]
I think if we implement #177 then we could store the appropriate construct queries for various REST APIs and people SPARQL query Twitter, arxiv, etc. without knowing the details.
